README.md

```diff
@@ -89,6 +89,9 @@ With more options:
 user2 = Plaid::User.exchange_token('public_token', account_id: '...', product: :auth)
 ```
 
+If you specify the `account_id` param, the returned user instance will have the
+`stripe_bank_account_token` attribute set.
+
```

Comment:
I admittedly don't have great context on how stripe works, what is `account_id` and how does it relate to `stripe_bank_account_token`?